src/app/services/theme/theme.service.ts

```diff
@@ -0,0 +1,202 @@
+import { DOCUMENT } from "@angular/common";
+import { Inject, Injectable } from "@angular/core";
+import { IS_SERVER_PLATFORM } from "src/app/app.helper";
+
+export const bawThemes = [
+  "highlight",
```

Comment:
Okay I see the confusion. A theme is a collection of colours, a palette. 

Multiple themes would be multiple different palettes. The scope of the required change is only one theme